src/tool/subcommands/api_cmd.rs

```diff
@@ -671,6 +717,237 @@ async fn compare_apis(
     run_tests(tests, &forest, &lotus, &config).await
 }
 
+async fn start_server(
+    snapshot_path_opt: Option<PathBuf>,
+    chain: NetworkChain,
+    rpc_port: u16,
+) -> anyhow::Result<()> {
+    println!("Configuring Node-Less RPC Server");
```

Comment:
surprising that we're not using `info!(..)` logs here

Reply:
changed to using logger now